src/integration/pages/onDemandAudioPage/canonicalTests.js

```diff
@@ -3,7 +3,7 @@ import {
   runCanonicalAnalyticsTests,
   runRadioScheduleTests,
   runCommonCrossPlatformTests,
-  runMediaPlayerEmbedLegacyTests,
```

Comment:
I'm not certain on this one. I checked where the function is used and found
this Media Asset Page [integration](https://github.com/bbc/simorgh/blob/71a5aca2e6a4af792bad7a49bdd7e536dd26f2c6/src/integration/pages/mediaAssetPage/crossPlatformTests.js#L13) test.
Shall we clean this up separately when these pages are removed?
Happy to remove it if we can establish that it's not needed for the Media Asset Page test suite.

Reply:
Let me have a quick dig about the code - I will confirm whether we still need it or not.

Reply:
@amoore108 could you assist us with this - are the legacy media player tests still needed for MAP, STY and PGL? I suppose this is technically scope creep, so should not be part of this ticket, but it would be good to clean up any media player tests which we no longer require.

Reply:
MAP, STY and PGL (not sure why PGL is in that code block as it shouldn't show media players) are all using the new MediaLoader on canonical.

They will still be referencing the old media player on AMP, but if these tests don't involve AMP then they could be deleted.

Leave them if you want though and I can deal with them in https://github.com/bbc/simorgh/pull/11994

Reply:
We're on a bit of a time crunch (30th September), so let's leave it for now and if you don't mind, this could be cleaned up as part of your PR?

Reply:
Yea of course, no bother at all. I can remove all the now legacy AVPlayer, MediaPlayer components etc and any related tests.

Reply:
Thanks both for helping reconcile where this change lies.